mobile_portfolio_upload_pulltest

marchelazo opened the pull request:
Trying to create a second branch (mobile_portfolio_upload_pulltest) with the correct files, comparing the changes now. In this PR it compared the main branch and the secondary branch, I also implemented the required Linters and made changes to the Index.html, styles.css & various others. These changes include basic corrections such as indents etc...

marchelazo committed:
mobile_portfolio_upload

Trying to create secondary branch with correct files

Abenezer-Tilahun reviewed (changes requested):
Hi @marchelazo    , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Good README
✔️ Good commit messages

## Required Changes ♻️
_Check the comments under the Review.

- [x] Kindly make sure your PR has a descriptive summary (details) describing introduced changes in more detail.

## Optional suggestions

- [x] Kindly Add descriptive README file to your project but remember to customize it to your project. please add your GitHub and LinkedIn username at the author section 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

marchelazo committed:
Updated README.md file 

The code editor made friendly suggestions so I made the changes for good practice.

HAMMAS-SALEEM commented on PersonalPortfolio/index.html:
```diff
@@ -0,0 +1,44 @@
+        <div class="connect">
+            <span>LET’S CONNECT</span>
+        </div>
+        <ul class="social-icons">
+            <li><img src="/PersonalPortfolio/imagenes/twiter.png" alt="twiter"></li>
+            <li><img src="/PersonalPortfolio/imagenes/Linkedin icon.png" alt="Linkedin"></li>
+            <li><img src="/PersonalPortfolio/imagenes/git.png" alt="git-ic"></li>
+            <li><img src="/PersonalPortfolio/imagenes/chill.png" alt="chill-ico"></li>
+        </ul>
```
- [ ] As the previous reviewer requested you to use anchor tags around your imag icons. Kindly use them as shown below template :arrow_down:

```
<ul>
  <li><a><img /></a></li>
  <li><a><img /></a></li>
  <li><a><img /></a></li>
  <li><a><img /></a></li>
</ul>
```

HAMMAS-SALEEM reviewed (changes requested):
# Required Changes :recycle:
![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f574770523531743971623739534e464f74732f67697068792e676966](https://user-images.githubusercontent.com/93039359/190567385-8e7bb3c0-7d3b-4d25-8385-61d173233786.gif)

## Hey @marchelazo :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title

Good job on implementing some of the requested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
